### PR TITLE
Fixing a link on the Transaction Signer view

### PR DESCRIPTION
**Pull request by ElliotFriend**
With the updated docs site, I'm willing to bet there are several more links that ought to be found/fixed, too. I'll try to search through the codebase and see if I can identify any more easy/quick fixes.

### CODE REVIEW

**Comment by ElliotFriend** [comment]
@briwylde08 Maybe you could look over the replacements that I've submitted, to see if there is a more fitting resource to link to? 😁

File: src/components/OperationPanes/SetOptions.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ export default function SetOptions(props) {
       label={
         <span>
           Set Flags{" "}
-          <HelpMark href="https://developers.stellar.org/docs/glossary/accounts/#flags" />
+          <HelpMark href="https://developers.stellar.org/docs/encyclopedia/flags" />
```
Comment (quietbits) [inline src/components/OperationPanes/SetOptions.js:68]: This link might have changed. https://developers.stellar.org/docs/glossary#flags works

**Review by quietbits — COMMENTED** [review]
Thanks for the PR, @ElliotFriend ! Could you please double-check if the links are still correct? I looked at a few, and some didn't work. :pray:

**Comment by ElliotFriend** [comment]
Sure thing! I can go back over it a little later today 👍

**Comment by ElliotFriend** [comment]
@quietbits I went through all the links I "corrected," and made sure everything is working. Let me know if there's anything else you'd like changed 👍🏻

**Review by quietbits — APPROVED** [review]
:rocket:

**Pull request merged**